### PR TITLE
Add note about increased window and interval can cause WiFi instability

### PR DESCRIPTION
**Pull request by bdraco**
## Description:

Add note about increased window and interval can cause WiFi instability
https://github.com/esphome/bluetooth-proxies/pull/71

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#<esphome PR number goes here>

## Checklist:

  - [ ] I am merging into `next` because this is new documentation that has a matching pull-request in [esphome](https://github.com/esphome/esphome) as linked above.  
    or
  - [x] I am merging into `current` because this is a fix, change and/or adjustment in the current documentation and is not for a new component or feature.

  - [ ] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

File: components/bluetooth_proxy.rst [hunk 1]
```diff
@@ -71,6 +73,8 @@ Below is a complete sample recommended configuration for a Bluetooth proxy. If y
 
 This configuration is for an Olimex ESP32-PoE-ISO board with an Ethernet connection to the network. If you use a different board, you must change the ``board`` substitution to match your board.
 
+For WiFi-based proxies, 1100ms can result in an unstable WiFi connection. Setting the default values for interval and window will usually resolve any instability.
+
```
Comment (emontnemery) [inline components/bluetooth_proxy.rst]: Not sure if we should repeat this?

Instead, maybe change the header to:
```
Complete sample recommended configuration for an ethernet connected Bluetooth proxy
-----------------------------------------------------------------------------------
```

File: components/bluetooth_proxy.rst [hunk 2]
```diff
@@ -61,6 +61,8 @@ To maximize the chances of catching advertisements of the sensors, you can set `
       scan_parameters:
         interval: 1100ms
         window: 1100ms
+        
+For WiFi-based proxies, 1100ms can result in an unstable WiFi connection. Setting the default values for interval and window will usually resolve any instability.
```
Comment (emontnemery) [inline components/bluetooth_proxy.rst]:
```suggestion
.. note::

    For WiFi-based proxies, changing the interval or window from their default values may result in an unstable WiFi connection. Keep the default values for interval and window will usually resolve any instability.
```

File: components/bluetooth_proxy.rst (hunk 1 above)
Comment (bdraco) [reply components/bluetooth_proxy.rst]: adjusted 👍

File: components/bluetooth_proxy.rst [hunk 3]
```diff
@@ -61,13 +61,16 @@ To maximize the chances of catching advertisements of the sensors, you can set `
       scan_parameters:
         interval: 1100ms
         window: 1100ms
+.. note::
+
+    For WiFi-based proxies, changing the interval or window from their default values may result in an unstable WiFi connection. Keep the default values for interval and window will usually resolve any instability.
```
Comment (bdraco) [inline components/bluetooth_proxy.rst]:
```suggestion
    For WiFi-based proxies, changing the interval or window from their default values may result in an unstable WiFi connection. Keeping the default values for interval and window will usually resolve any instability.
```

File: components/bluetooth_proxy.rst [hunk 4]
```diff
@@ -61,13 +61,16 @@ To maximize the chances of catching advertisements of the sensors, you can set `
       scan_parameters:
         interval: 1100ms
         window: 1100ms
+.. note::
+
+    For WiFi-based proxies, changing the interval or window from their default values may result in an unstable WiFi connection. Keeping the default values for interval and window will usually resolve any instability.
```
Comment (bdraco) [inline components/bluetooth_proxy.rst]:
```suggestion
    For WiFi-based proxies, changing the `interval` or `window` from their default values may result in an unstable WiFi connection. Using the default values for `interval` and `window` will usually resolve any instability.
```

File: components/bluetooth_proxy.rst [hunk 5]
```diff
@@ -61,13 +61,16 @@ To maximize the chances of catching advertisements of the sensors, you can set `
       scan_parameters:
         interval: 1100ms
         window: 1100ms
+.. note::
+
+    For WiFi-based proxies, changing the `interval` or `window` from their default values may result in an unstable WiFi connection. Using the default values for `interval` and `window` will usually resolve any instability.
```
Comment (bdraco) [inline components/bluetooth_proxy.rst]:
```suggestion
    For WiFi-based proxies, changing the ``interval`` or ``window`` from their default values may result in an unstable WiFi connection. Using the default values for ``interval`` and ``window`` will usually resolve any instability.
```

**Review by emontnemery — APPROVED** [review]

**Pull request merged**